sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -1,16 +1,19 @@
 import { IsNotEmpty, IsOptional } from "class-validator";
-export class StudentConfirmInfo {
+export class StudentInfo {
   firstName: string;
   lastName: string;
-  addressLine1: string;
-  addressLine2: string;
-  city: string;
-  provinceState: string;
-  country: string;
-  postalCode: string;
+  email: string;
   gender: string;
-  phoneNumber: string;
   dateOfBirth: Date;
+  contact = new (class contact {
```

Comment:
I would prefer to not use syntax like "new (class contact".
I believe that just property = { ... } would be clear.